mozdownload/scraper.py

```diff
@@ -800,6 +800,7 @@ def get_build_info(self):
         if not self.locale_build:
             self.builds, self.build_index = self.get_build_info_for_index(
                 self.build_index)
+            self.timestamp = self.builds[self.build_index]
```

Comment:
If there is no build available, this would raise an exception. Should we allow this? And if yes, how do we handle that?

Reply:
My assumption was that https://github.com/Nebelhom/mozdownload/blob/7bdceeacc835d958be17ef802eb21e9f1569b222/mozdownload/scraper.py#L908 would raise an issue if there is no build available, so it would never reach this assignment...

Reply:
Oh, right. So that's fine then. Can you add a comment which explains why we set the timestamp here?